README.md

```diff
@@ -0,0 +1,184 @@
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+ 
+
+- **Create a budget app**
+- **register and log in, so that the data is private to them.** 
+- **introduce new transactions associated with a category.**
+- **see the money spent on each category.**
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+ 
+
+- [Live Demo Link]()
+
+<p align="right"><a href="#readme-top">back to top</a></p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+**To get a local copy up and running, follow these steps.**
+1. Open terminal or cmd on you local machine and navigate to your desired directory.
+2. **Run**    *'git clone clone https://github.com/ecekpo/Budget-app.git'*
+3. **Run**   *'code .'*
+
+### Prerequisites
+
+
+In order to run this project you need **[describe prerequisites]**:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/ecekpo/Budget-app.git
+  cd Budget_app
+```
+
+### Install
+
+Install this project with **[package manager]**:
+
+```sh
+  cd Budget_app
+  gem install
+  rails db:migrate
+  rails db:migrate RAILS_ENV=test
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## Author
+
+👤 **Emem Ekpo**
+
+* GitHub: [@githubhandle](https://github.com/emem-ekpo)
+* Twitter: [@twitterhandle](https://twitter.com/ememcookey)
+* LinkedIn: [@LinkedIn](https://www.linkedin.com/in/emem-ekpo)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Features of this project.
+
+- [ ] **User Authentication**
+- [ ] **CRUD Operations**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Credit
+   I give credit to the original design idea by Gregoire Vella on Behance.
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] [REQUIRED] Good job so far! It woud be proffesional if you create `MIT` license for your application after create `MIT` remove the `line 184` 🙏🏼